README.md

```diff
@@ -12,7 +12,7 @@ Binaries for handling SQS Dead Letter Queues:
 
 ### sqs-dead-letter-requeue
 ```sh
-go build sqs-dead-letter-requeue.go
+go build -o bin/sqs-dead-letter-requeue sqs-dead-letter-requeue/main.go
```

Comment:
By convention, this should be `cmd/sqs-dead-letter-requeue/main.go`. Then add more tools to `cmd/` as you go.